packages/jest-core/src/watch.ts

```diff
@@ -228,6 +231,22 @@ export default async function watch(
   let shouldDisplayWatchUsage = true;
   let isWatchUsageDisplayed = false;
 
+  // Initialize Vite dev server manager
+  let viteDevServerManager: ViteDevServerManager | null = null;
```

Comment:
I wonder if the implementation for Vite dev server should put here? Can you investigate where we can put the code? I think in the case of `vitest`, the dev server is invoked by running `vitest` CLI right? Maybe we should somehow invoke from CLI or `jest-core`?